.github/workflows/pr.yaml

```diff
@@ -26,14 +26,21 @@ jobs:
   plugins_test:
     name: Plugins Test Runner
     runs-on: ubuntu-latest
-    timeout-minutes: 5
+    timeout-minutes: 20
     strategy:
       matrix:
         linter-version: [KnownGoodVersion, Latest]
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Cache tool downloads
+        uses: actions/cache@v3
+        with:
+          path: /tmp/plugins_testing_download_cache
```

Comment:
Where does this get wired into trunk? Inside trunk-io/action?

Reply:
It's built-in in driver.ts. Sets `TRUNK_DOWNLOAD_CACHE` to `plugins_testing_download_cache`. The trunk-io/action has its own caching that is auto-handled that is in a different location. We don't have any nightlies yet here, so the clearing logic you mentioned doesn't apply here.

This should probably be wired in more clearly or from a higher level. No immediate ideas, but we can address in a follow-up